verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -138,15 +140,50 @@ void IndexingFactsTreeExtractor::ExtractModule(
     Visit(module_item_list);
   }
 
-  facts_tree_context_.back()->NewChild(module_node);
+  facts_tree_context_.top().NewChild(module_node);
 }
 
 void IndexingFactsTreeExtractor::ExtractModuleHeader(
     const verible::SyntaxTreeNode& node) {
```

Comment:
Name the parameter with the type of node that you expect to be passed in. e.g. `module_declaration_node`.
(I know I should do this better throughout the codebase.)

Same elsewhere a parameter is named `node` or `leaf`.